ios/BetterRailWidget/Live Activity/Views/DynamicIsland/LeadingView.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+import SwiftUI
+import WidgetKit
+import ActivityKit
+
+
+struct LeadingView: View {
+  var vm: ActivityViewModel
+  
+  var headerStation: String {
+    if (vm.status == .waitForTrain || vm.status == .inExchange) {
+      return vm.train.originStationName
+    }
+    
+    return vm.train.destinationStationName
```

Comment:
If my first train ends in Savidor, and my journey is to Jerusalem, Will the first text be "headed to Savidor" or "headed to Jerusalem"?

Reply:
Headed to Savidor. 

I've pushed a fix so it'll display the final destination